cluster/images/hyperkube/daemonsets/flannel-ds.yaml

```diff
@@ -0,0 +1,46 @@
+---
+  apiVersion: extensions/v1beta1
+  kind: DaemonSet
+  metadata:
+    name: k8s-flannel
+    namespace: kube-system
+    labels:
+      name: k8s-flannel
+      version: v1
+      kubernetes.io/cluster-service: "true"
+  spec:
+    template:
+      metadata:
+        labels:
+          name: k8s-flannel
+          version: v1
+          kubernetes.io/cluster-service: "true"
+      spec:
+        hostNetwork: true
+        containers:
+          -
+            securityContext:
+              privileged: true
+            name: k8s-flannel
+            image: quay.io/coreos/flannel:0.5.5
+            command:
+              - /opt/bin/flanneld
+              - --ip-masq=true
+              - --iface=eth0
```

Comment:
Does anyone have an idea how we could configure different interfaces for each node (if necessary). Right now the `daemonset` definition is global for the cluster. That would be node-specific configuration, however.